pages.ru/linux/ispell.md

```diff
@@ -0,0 +1,16 @@
+# ispell
+
+> Интерактивная проверка орфографии.
+> Больше информации: <https://www.cs.hmc.edu/~geoff/ispell.html>.
+
+- Начать интерактивную сессию:
+
+`ispell`
+
+- Проверить на ошибки указанный файл и интерактивно применить исправления:
+
+`ispell {{путь/до/файла}}`
+
+- Отобразить версию:
+
+`ispell -v`
```

Comment:
Long option should be used.

Reply:
This command too doesn't support longer arguments, The link can be updated to a more specific one (https://www.cs.hmc.edu/~geoff/ispell-man.html) I will do it in a new PR.